tests/cibyl/unit/sources/elasticsearch/test_api.py

```diff
@@ -348,6 +348,31 @@ def test_get_tests_filter_by_tests_name(self: object,
             1
         )
 
+    @patch.object(ElasticSearch, '_ElasticSearch__query_get_hits')
```

Comment:
Something is wrong with the design if you have to mock a private function. Other languages will not even allow you to. On any case, not much to do now about it.

Reply:
That's a fair point, but since this would require changing the testing for the entire module, I'll leave it for another change.